src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -75,6 +76,7 @@ const KeyboardAwareScrollView = forwardRef<
       children,
       bottomOffset = 0,
       disableScrollOnKeyboardHide = false,
+      enableKeyboardHandling = true,
```

Comment:
Also, please add this property to snapshot tests (in example and FabricExample apps)